Composer/packages/extensions/visual-designer/src/components/nodes/templates/ListOverviewCard.tsx

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { FunctionComponent } from 'react';
+
+import { InitNodeSize } from '../../../constants/ElementSizes';
+import { ElementIcon } from '../../../utils/obiPropertyResolver';
+import { Icon } from '../../decorations/icon';
+import { SingleLineDiv } from '../../elements/styledComponents';
+
+const boxWidth = InitNodeSize.width;
+const boxHeight = InitNodeSize.height;
+const headerHeight = InitNodeSize.height / 2;
+const contentHeight = boxHeight - headerHeight;
+
+const containerStyle = {
+  width: boxWidth,
+  minHeight: boxHeight,
+  fontSize: '12px',
+  cursor: 'pointer',
+  overflow: 'hidden',
+  backgroundColor: 'white',
+  borderRadius: '2px 2px 0 0',
+  boxShadow: '0px 2px 8px rgba(0, 0, 0, 0.1)',
+};
+
+interface NodeProps {
+  header: string;
+  corner?: any;
+  label?: any;
+  icon?: string;
+  iconSize?: number;
+  styles?: object;
+  nodeColors: { [key: string]: any };
+  onClick?: () => void;
+  children?: any;
+}
+export const ListOverviewCard: FunctionComponent<NodeProps> = ({
```

Comment:
`ListOverviewCard` should at least consume the `ListOverview` FC component, otherwise it's not a 'ListOverview' card.